web-server/package.json

```diff
@@ -123,7 +123,7 @@
     "eslint-config-next": "13.5.6",
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-prettier": "^5.0.1",
-    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react": "^7.37.2",
```

Comment:
Is this change actually needed?

If not, you might be able to reduce the size of your PR significantly.

Reply:
@not-meet reminder ☝🏽

Reply:
@jayantbh hi really sorry for the delay ill fix this asap, actually this happened  back when the pr was not able to merge because of linting then i used pre-commit to fix the issue and it gave me an error and a command to fix the error and after that when i ran pre-commit this issue happened... ill fix this as soon as possible.. :-)

Reply:
@jayantbh hi.. i made the eslint-plugin- react changes as they were also ran pre-commit run all files to make sure the linting problem dose not happen again...i hope this will solve all the issues while merging the pr :-)